docker/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.6'
+services:
+  elasticsearch-01:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.16.2
+    container_name: elasticsearch-01
+    environment:
+      - node.name=elasticsearch-01
+      - cluster.name=ctautocomplete-es
+      - cluster.initial_master_nodes=elasticsearch-01
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - data01:/usr/share/elasticsearch/data
+      - ./config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+    networks:
+      - elastic
+
+  ct-autocomplete:
+    image: registry.ctcue.com/ctautocomplete:es7
+    container_name: ct-autocomplete
+    ports:
+      - 4080:4080
+    volumes:
+      - ./config/local_config.json:/app/config/local_config.json
```

Comment:
🏖️ If it's easier, we could replace the config with `env` variables too... the only config we have is the elasticsearch host really

Reply:
We dont need much indeed, however now we have the config file on the host that we can check and edit. Shouldn't require much though :p